### ISSUE TITLE
Load error when `erb` gem is not loaded elsewhere

### ISSUE THREAD

**Issue by muellerj** [MiniProfiler/rack-mini-profiler#530]
Including `rack-mini-profiler` inside an app, which doesn't `require "erb"` leads to an error:

```
NameError
uninitialized constant Rack::MiniProfiler::TimerStruct::Sql::ERB

/usr/lib/ruby/gems/3.0.0/gems/rack-mini-profiler-3.0.0/lib/mini_profiler/timer_struct/sql.rb:41:in `initialize'
```

The line in question reads

``` ruby
formatted_command_string: query ? ERB::Util.html_escape(query) : nil,
```

Maybe add a `require "erb"` at the top of  `lib/mini_profiler/timer_struct/sql.rb`? Or use `CGI::escape` directly?

**Comment by OsamaSayegh** [comment]
> Maybe add a `require "erb"` at the top of `lib/mini_profiler/timer_struct/sql.rb`?

Yes, we should require `ERB` explicitly, but maybe a better place for it would at the top of `lib/rack-mini-profiler` next to the other standard libraries required for Mini Profiler?

https://github.com/MiniProfiler/rack-mini-profiler/blob/1ed6d97d979c3a1819cc69daa1d0475a17d14a54/lib/rack-mini-profiler.rb#L1-L7

Would you like to take a shot at this and send a PR?

**Comment by OsamaSayegh** [comment]
Fixed in #531.

Thanks again @muellerj!
